src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -199,6 +203,11 @@ public static FeatureChange remove(final AtlasEntity reference, final Atlas atla
         }
     }
 
+    public String getFeatureChangeId()
```

Comment:
Let's do "Identifier", we generally avoid abbreviations.

Reply:
Ok.